test/resources/test_reporting_unit_case_group_status.py

```diff
@@ -0,0 +1,178 @@
+import json
+import unittest
+
+import requests_mock
+
+from ras_backstage import app
+from ras_backstage.resources.case.reporting_unit_case_group_status import filter_statuses
+
+url_get_party_by_ru_ref = f'{app.config["RAS_PARTY_SERVICE"]}party-api/v1/parties/type/B/ref/12345'
```

Comment:
This is fine here for now but it's worth mentioning that I really like what @LJBabbage did with these mock urls/data moving them into their own file. Might be worth the refactor at some point

Reply:
Yeah it could make it nicer. I did give it a go but quickly noticed they use different params in the URL so created https://trello.com/c/lOFq7CNV/72-refactor-common-urls-used-in-the-python-flask-tests-into-common-testurlspy-file